docs/release-notes/release-notes-0.15.0.md

```diff
@@ -164,6 +164,9 @@ from occurring that would result in an erroneous force close.](https://github.co
 * [Taproot wallet inputs can also be used to fund
   channels](https://github.com/lightningnetwork/lnd/pull/6521)
 
+* [Fixed a wrong unit used in the field `push_amount_sat` from the rpc
```

Comment:
does it need a note if the issue wasn't in any prior release?

Reply:
make sense, will remove!